blog/2022-06-30-paralus-quickstart-aks/index.md

```diff
@@ -0,0 +1,200 @@
+---
+slug: paralus-quickstart-aks
+title: "Tutorial: How to set up Paralus on Azure Kubernetes Service - AKS"
+authors: [atul]
+tags: [azure, aks, tutorial]
+---
+
+The [previous blog post](/blog/eks-quickstart), was about deploying Paralus to Amazon's Elastic Kubernetes Service (EKS).
+
+In this blog post, we'll take you through the steps to setup an Azure Kubernetes Services (AKS) cluster on a custom domain and deploy Paralus on it.
+
+<!--truncate-->
+
+**Table Of Content:**
+
+- [Pre Requisites](#pre-requisites)
+- [Creating AKS Cluster](#creating-aks-cluster)
+- [Installing Paralus](#installing-paralus)
+- [Configuring DNS Settings](#configuring-dns-settings)
+  - [Accessing The Dashboard](#accessing-the-dashboard)
+  - [Importing Existing Cluster](#importing-existing-cluster)
+
+## Pre Requisites
+
+To setup Paralus on Azure Kubernetes Service (AKS) there are a few prerequisites:
+
+- An Azure Account - _you can [register for a Free Azure account](https://azure.microsoft.com/en-in/free/) if you don't have one_
+- A Domain Name - _with permission to manage DNS settings._
+- Helm
+
+The logical steps would start with setting up a Kubernetes cluster on AKS. Deploying Paralus via helm charts followed by configuring DNS for your domain to work with Paralus. Lastly, logging into Paralus and importing a Kubernetes cluster.
+
+## Creating AKS Cluster
+
+One of the good things when it comes to creating an AKS cluster is the various options it gives you to create a cluster:
+
+- [Using the Azure CLI](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli)
+- [Using the Azure PowerShell](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+- [Using the Azure Portal](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-powershell)
+- [Using the ARM template](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-rm-template)
+
+You are free to choose any option that you want, we used the Azure Portal to create one.
+
+> Note: Choose a virtual machine with atleast 4 vCPUs and 16 GB of RAM. Refer to the list of [Azure VMs](https://azure.microsoft.com/en-in/pricing/details/virtual-machines/linux/).
+
+After the cluster is created, start the cluster and connect to it. If you've created a cluster using the Azure portal, you can use the Azure Cloud Shell to connect to the cluster.
+
+## Installing Paralus
+
+Clone the [paralus helm repository](https://github.com/paralus/helm-charts)
+
+1. Add helm repo
+
+   `helm repo add paralus https://paralus.github.io/helm-charts`
+
+2. Create `values.aks.yaml`
+
+   ```yaml
+   ingress:
+     enabled: false
```

Comment:
default is false. So this is not required.

Reply:
Removed from this as well a EKS blog post.